packages/mantine/src/components/collection/CollectionItem.tsx

```diff
@@ -8,20 +8,23 @@ import {ActionIcon} from '../action-icon';
 import {useCollectionContext} from './CollectionContext';
 
 interface CollectionItemProps extends CollectionItemSharedProps {
+    data_index: number;
```

Comment:
Just noticed that this prop is duplicated. CollectionItemProps extends CollectionItemSharedProps, so I would remove this line and keep only in the CollectionItemSharedProps interface